Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.12.2-alpine3.19
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
```

Comment:
There is a typo in the environment variable name `PYTHOUNNBUFFERED`. It should be `PYTHONUNBUFFERED` to ensure Python output is sent straight to the terminal without being buffered.